liubov.tarasenko/homework3/prime.rb

```diff
@@ -0,0 +1,25 @@
+class PrimeMatrix
+  def initialize(max)
+    @max = max
+  end
+  
+  def prime_arr
+    base = [2,3,5,7].select { |x| x <= @max }    
+    rest = (2..@max).to_a.reject { |x| base.any? { |b| x % b == 0 } }
```

Comment:
```suggestion
    rest = (2..@max).reject { |x| base.any? { |b| x % b == 0 } }
```

Usage of `to_a` is redundant here. 
`Range` class includes almost all `Enumerable` actions.